### ISSUE TITLE
EN_getMatchingEnts api change in 15.0-190708dev

### ISSUE THREAD

**Issue by jacobmerson** [SCOREC/core#242]
The following error is found compiling core with the version of simmodsuite on DCS. According to the documents adding an integer argument of 0 to the end should maintain the current functionality.

```c++
/gpfs/alpine/scratch/jmerson/csc370/core/apf_sim/apfSIM.cc:1045:18: error: 
      no matching function for call to
      'EN_getMatchingEnts'
      pPList l = EN_getMatchingEnts(ent, NULL);
                 ^~~~~~~~~~~~~~~~~~
/gpfs/alpine/scratch/jmerson/csc370/simmetrix/simmodsuite/15.0-190708dev/include/SimModel.h:103:21: note: 
      candidate function not viable: requires 3
      arguments, but 2 were provided
Model_EXPORT pPList EN_getMatchingEnts(pEn...
                    ^
1 error generated.
```

**Comment by cwsmith** [comment]
Thank you.

The branch with a fix, and continued support for SimModSuite 14, is here:

https://github.com/SCOREC/core/tree/simmodsuite15

The branch won't be merged into develop until we hear back from Simmetrix on a problem with the pumi `generate` utility.

Let's leave this issue open until we hear back from Simmetrix.

**Comment by cwsmith** [comment]
The compatibility problem has been fixed with SimModSuite 14.0-190928 and  15.0-190821dev (both now installed on scorec workstations).  The `simmodsuite15` branch has been merged into develop.

**Comment by usmanriaz07** [comment]
Thank you for letting me know.

Regards
Usman
